model/src/main/java/org/cbioportal/genome_nexus/model/MutationAssessor.java

```diff
@@ -2,43 +2,46 @@
 
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
 
-@Document(collection = "mutation_assessor")
+@Document(collection = "mutation_assessor.annotation")
 public class MutationAssessor
 {
     @Id
     private String input;
 
+    private String hgvs;
+
     private String hugoSymbol;
 
     private String referenceGenomeVariant;
     private String referenceGenomeVariantType;
 
     private String functionalImpact;
-    private double functionalImpactScore;
+    private Double functionalImpactScore;
```

Comment:
Using primitive types causes database mapping problems if the value is null. So, changed all `double`s to `Double`, and all `int`s to `Integer`.